cms/djangoapps/contentstore/tasks.py

```diff
@@ -261,7 +262,7 @@ def export_olx(self, user_id, course_key_string, language):
         LOGGER.exception(u'Error exporting course %s', courselike_key, exc_info=True)
         if self.status.state != UserTaskStatus.FAILED:
             self.status.fail({'raw_error_msg': text_type(exception)})
-        return
+        raise
```

Comment:
Should all `LOGGER.exception` calls in this file now be replaced with `raise`?  As a [recent celery-utils PR](https://github.com/edx/edx-celeryutils/pull/14) now logs them for `LoggedTask`s?

Reply:
Also, is changing this from `return` to `raise` going to cause any unexpected issues?